### PR TITLE
SEO-179381-MAUI-Shimmer-External-Linking

### PR DESCRIPTION
**Pull request by berylouma**
|Title|Description|
|--|--|
|Task Category| .NET MAUI Shimmer|
|Content Task Link/Mail Screenshot||
|UX task| NA|
|External Linking PR link| Ext - https://github.com/SyncfusionExamples/Getting-started-with-the-.NET-MAUI-Shimmer-control/pull/1|
|Ticket/Task link| https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/179381/|
|Excel/SharePoint link| https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/NET-MAUI-Shimmer---Money-Keywords.aspx|
 

Worked on 17 keywords in the .NET MAUI Shimmer and am keeping track of all keyword details in the link below:-
[https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/Keyword-Plan---.NET-MAUI-Shimmer.aspx]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,63 @@
...
+The [.NET MAUI Shimmer](https://www.syncfusion.com/maui-controls/maui-shimmer) control can be used to improve the responsiveness of an application by showing a modern shimmer effect when data is being loaded in the background. A quick-start example to help you how to add Syncfusion’s .NET MAUI Shimmer control to a .NET MAUI  project and work with the shimmer views and customization. The Syncfusion .NET MAUI Shimmer control increases an application’s responsiveness by displaying a shimmer effect while the content is loading in the background. There are several view types in the .NET MAUI Shimmer such as article, profile, video, and more. The control allows users to customize the wave’s color, width, direction, and animation duration.
```
Comment (Sharly-Atieno) [inline README.md]: Add utm parameter after the link, and recheck the content

File: README.md [hunk 2]
```diff
@@ -1,16 +1,63 @@
...
+## Features and Benefits
+
+### Built-in types
+Use different built-in shimmer types, such as article, circle persona, feed, profile, square persona, shopping, and video, in your application to create interesting shimmer effects.
+
+### Custom view
+Custom shimmer patterns in a variety of forms provide an alternative to the built-in shimmer types.
+
+### Customization
+The appearance of a .NET MAUI Shimmer control is highly customizable.
```
Comment (Sharly-Atieno) [inline README.md]: Interlink UG links where possible..

File: README.md [hunk 3]
```diff
@@ -1,16 +1,63 @@
...
+
+[Documentation](https://help.syncfusion.com/maui/shimmer/getting-started?utm_source=github&utm_medium=listing&utm_campaign=maui-shimmer-github-samples)
+
+[View Demos](https://github.com/SyncfusionExamples/Getting-started-with-the-.NET-MAUI-Shimmer-control?utm_source=github&utm_medium=listing&utm_campaign=maui-shimmer-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-shimmer-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-shimmer-github-samples)
+
+[Online example](https://github.com/syncfusion/maui-demos/?utm_source=github&utm_medium=listing&utm_campaign=maui-shimmer-github-samples)
```
Comment (Sharly-Atieno) [inline README.md]: updated the correct link here

File: README.md [hunk 4]
```diff
@@ -1,16 +1,63 @@
...
+
+[Knowledge base](https://support.syncfusion.com/kb/cross-platforms/section/1255?utm_source=github&utm_medium=listing&utm_campaign=maui-shimmer-github-samples)
```
Comment (Sharly-Atieno) [inline README.md]: updated the specific KB link for .NET MAUI Shimmer

File: README.md [hunk 5]
```diff
@@ -1,16 +1,63 @@
...
+
+
+## About Syncfusion .NET MAUI Components
+
+Syncfusion's [.NET MAUI Shimmer](https://www.syncfusion.com/maui-controls/maui-shimmer?utm_source=github&utm_medium=listing&utm_campaign=maui-shimmer-github-samples) library is the only suite that you will ever need to build an application since it contains over 40 high-performance, lightweight, modular, and responsive UI controls in a single package. In addition to Shimmer, we provide popular .NET MAUI Controls such as [DataGrid](https://www.syncfusion.com/maui-controls/maui-datagrid?utm_source=github&utm_medium=listing&utm_campaign=maui-shimmer-github-samples), [Charts](https://www.syncfusion.com/maui-controls/maui-cartesian-charts?utm_source=github&utm_medium=listing&utm_campaign=maui-shimmer-github-samples), [ListView](https://www.syncfusion.com/maui-controls/maui-listview?utm_source=github&utm_medium=listing&utm_campaign=maui-shimmer-github-samples), and [Excel Library](https://www.syncfusion.com/document-processing/excel-framework/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-shimmer-github-samples).
```
Comment (Sharly-Atieno) [inline README.md]: Recheck on the content..

File: README.md (hunk 3 above)
Comment (berylouma) [reply README.md]: Updated

File: README.md (hunk 4 above)
Comment (berylouma) [reply README.md]: Modified

File: README.md (hunk 5 above)
Comment (berylouma) [reply README.md]: Corrected

File: README.md (hunk 2 above)
Comment (berylouma) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (berylouma) [reply README.md]: Updated

**Review by Sharly-Atieno — APPROVED** [review]
Approved

File: README.md [hunk 6]
```diff
@@ -1,16 +1,61 @@
...
+## Related links
+[Learn More about .NET MAUI Shimmer](https://www.syncfusion.com/maui-controls/maui-shimmer?utm_source=github&utm_medium=listing&utm_campaign=maui-shimmer-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-shimmer-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/teamlicense?utm_source=github&utm_medium=listing&utm_campaign=maui-shimmer-github-samples)
+
+[Documentation](https://help.syncfusion.com/maui/shimmer/getting-started?utm_source=github&utm_medium=listing&utm_campaign=maui-shimmer-github-samples)
+
+[View Demos](https://github.com/SyncfusionExamples/Getting-started-with-the-.NET-MAUI-Shimmer-control?utm_source=github&utm_medium=listing&utm_campaign=maui-shimmer-github-samples)
```
Comment (Girija-Rajendran) [inline README.md]: need to change view demos and online example link

Comment (berylouma) [reply README.md]: Changed

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**